tests/Doctrine/Tests/ORM/Functional/Ticket/GH8443Test.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\ORM\Functional\Ticket;
+
+use Doctrine\Orm\Query;
+use Doctrine\Tests\Models\Company\CompanyManager;
+use Doctrine\Tests\Models\Company\CompanyPerson;
+use Doctrine\Tests\OrmFunctionalTestCase;
+
+use function count;
+
+final class GH8443Test extends OrmFunctionalTestCase
+{
+    /**
+     * @group GH-8443
+     */
+    public function testJoinRootEntityWithForcePartialLoad(): void
```

Comment:
Running that test by itself fails with `SQLSTATE[HY000]: General error: 1 no such table: company_persons`.
Tests should be independent from each other